test/rooms_test.js

```diff
@@ -1702,6 +1702,15 @@ describe("/rooms", function() {
           });
         });
 
+        it("should log the roomConnectionId", function(done) {
+          updateStateRoom(hawkCredentials, roomToken).end(function(err) {
+            if (err) throw err;
+            expect(logs).to.length(3);
+            expect(logs[2].roomConnectionId).to.not.be.undefined;
+            done();
+          });
+        });
```

Comment:
I think I'd expect to see the rainy-day case (i.e., user not in room) tested here as well.